pkg/service/account/service_application.go

```diff
@@ -0,0 +1,232 @@
+package account
+
+import (
+	"fmt"
+
+	"github.com/ans-group/sdk-go/pkg/connection"
+)
+
+// GetApplications retrieves a list of applications
+func (s *Service) GetApplications(parameters connection.APIRequestParameters) ([]Application, error) {
+	return connection.InvokeRequestAll(s.GetApplicationsPaginated, parameters)
+}
+
+func (s *Service) GetApplicationsPaginated(parameters connection.APIRequestParameters) (*connection.Paginated[Application], error) {
+	body, err := s.getApplicationsPaginatedResponseBody(parameters)
+	return connection.NewPaginated(body, parameters, s.GetApplicationsPaginated), err
+}
+
+func (s *Service) getApplicationsPaginatedResponseBody(parameters connection.APIRequestParameters) (*connection.APIResponseBodyData[[]Application], error) {
+	body := &connection.APIResponseBodyData[[]Application]{}
+
+	response, err := s.connection.Get("/account/v1/applications", parameters)
+	if err != nil {
+		return body, err
+	}
+
+	return body, response.HandleResponse(body, nil)
+}
+
+// GetApplication retrieves a single application by id
+func (s *Service) GetApplication(appID string) (Application, error) {
+	body, err := s.getApplicationResponseBody(appID)
+
+	return body.Data, err
+}
+
+func (s *Service) getApplicationResponseBody(appID string) (*connection.APIResponseBodyData[Application], error) {
+	body := &connection.APIResponseBodyData[Application]{}
```

Comment:
Just a suggestion - there's helpers available in the `connection` package which will help remove a lot of the boiler plate. See the PSS `GetCase` method here for an example:

https://github.com/ans-group/sdk-go/blob/master/pkg/service/pss/service_case.go#L32-L38